scripts/beachball/src/register.js

```diff
@@ -5,7 +5,6 @@ exports.register = register;
  */
 function register() {
   if (process.env.NODE_ENV !== 'test') {
-    // @ts-expect-error - ts-node/register doesn't provide types so this would error
```

Comment:
how come this is not needed anymore ?

Reply:
fixed, guess something was wrong on my side with ts language server :)

Reply:
gives a warning instead of an error, leading to unused ts suppression error